core/delegate/delegate.go

```diff
@@ -27,6 +27,7 @@ func NsExec(pid int, event *revents.Event) (int, string, map[string]interface{},
 
 	envmap := map[string]string{}
 	file, fileErr := os.Open(fmt.Sprintf("/proc/%v/environ", pid))
+	defer file.Close()
```

Comment:
This line needs to be after you check fileErr or you might get a panic referencing a nil pointer